src/crud/useRole.ts

```diff
@@ -66,10 +66,20 @@ export const useRole = () => {
 
   const getMbrRole = (memberRecs: Record[]) => {
     if (memberRecs.length === 1) {
-      const roleId = related(memberRecs[0], 'role');
+      var roleId = related(memberRecs[0], 'role');
+      if (!roleId) {
+        roleId = getRoleId(RoleNames.Admin);
```

Comment:
Would we want to default to Editor if no role is found?

Reply:
No, because this happens when we create a project, so we'd want to be admin